src/main/java/org/junit/runner/notification/SynchronizedRunListener.java

```diff
@@ -43,6 +43,37 @@ public void testRunFinished(Result result) throws Exception {
         }
     }
 
+    /**
+     * Synchronized decorator for {@link RunListener#testSuiteStarted(Description)}.
+     * @param description the description of the test suite that is about to be run
+     *                    (generally a class name)
+     * @throws Exception if any occurs.
+     * @since 4.13
+     *
+     * @see RunListener#testSuiteStarted(Description)
```

Comment:
Instead of using `@see` use `{@inheritDoc}`. See http://docs.oracle.com/javase/1.5.0/docs/tooldocs/windows/javadoc.html#{@inheritDoc} and http://tornorbye.blogspot.com/2005/02/inheriting-javadoc-comments.html